pkg/tenant/projects.go

```diff
@@ -10,29 +10,68 @@ import (
 	"github.com/rs/zerolog/log"
 )
 
+// TenantProjectList retrieves all projects assigned to a tenant
+// and returns a 200 OK response.
+//
+// Route: //tenant/:tenantID/projects
 func (s *Server) TenantProjectList(c *gin.Context) {
-	c.JSON(http.StatusNotImplemented, "not implemented yet")
+	var (
+		err error
+	)
+	// Get the project's tenant ID from the URL and return a 400 response
+	// if the tenant ID is not a ULID.
+	var tenantID ulid.ULID
+	if tenantID, err = ulid.Parse(c.Param("tenantID")); err != nil {
+		log.Error().Err(err).Msg("could not parse tenant ulid")
+		c.JSON(http.StatusBadRequest, api.ErrorResponse("could not parse tenant ulid"))
+		return
+	}
+
+	// Get projects from the database and return a 500 response
+	// if not successful.
+	if _, err := db.ListProjects(c.Request.Context(), tenantID); err != nil {
```

Comment:
It looks like `ListProjects` will hand us back a slice of projects (type `[]*Project`), which I think we want to assign to a variable rather than to a `_` so that we can capture the database response, and then assign it to `out.TenantProjects`.